.github/workflows/ci-nightly.yml

```diff
@@ -46,6 +46,7 @@ jobs:
     with:
       arch: ${{ matrix.arch }}
       full_arch: ${{ matrix.full_arch }}
+      cmake-parameters:
```

Comment:
If you make `cmake-parameters` not required, and set a default of `""`, you can leave this one out :)